backends-common/opensearch/src/main/java/org/apache/james/backends/opensearch/OpenSearchIndexer.java

```diff
@@ -90,7 +90,7 @@ public Mono<BulkResponse> update(List<UpdatedRepresentation> updatedDocumentPart
             op -> op.update(idx -> idx
                 .index(aliasName.getValue())
                 .id(updatedDocumentPart.getId().asString())
-                .document(Collections.singletonMap("doc", new RawValue(updatedDocumentPart.getUpdatedDocumentPart())))
```

Comment:
maybe check style error here
unused import `Collections` has not been removed

Reply:
Yep yep well caught